src/util/psbt/error.rs

```diff
@@ -100,7 +100,7 @@ impl fmt::Display for Error {
                 write!(f, "Preimage {:?} does not match {:?} hash {:?}", preimage, hash_type, hash )
             }
             Error::MergeConflict(ref s) => { write!(f, "Merge conflict: {}", s) }
-            Error::ConsensusEncoding => f.write_str("bitcoin consensus encoding error"),
+            Error::ConsensusEncoding => f.write_str("bitcoin consensus or BIP-174 encoding error"),
```

Comment:
Is this still the case after introducing `PsbtParseError`?

Reply:
Yes, since it is about BIP-174 consensus encoding (PSBT binary encoding) and has nothing to do with Base64 or `FromStr` implementation